client/me/help/help-contact/index.jsx

```diff
@@ -294,7 +294,7 @@ class HelpContact extends React.Component {
 		if ( site || userDeclaredUrl ) {
 			const siteUrl = userDeclaredUrl ? userDeclaredUrl.trim() : site.URL;
 
-			blogHelpMessage = this.translateForForums( 'Site: %s.', {
+			blogHelpMessage = this.translateForForums( 'Site: %s', {
```

Comment:
This addresses the request made in https://github.com/Automattic/wp-calypso/pull/52979#issuecomment-888109865